src/libs/actions/Task.ts

```diff
@@ -1287,8 +1289,11 @@ function getFinishOnboardingTaskOnyxData(taskName: keyof OnyxTypes.IntroSelected
 
     return {};
 }
-function completeTestDriveTask() {
-    getFinishOnboardingTaskOnyxData('viewTour');
+function completeTestDriveTask(shouldUpdateSelfTourViewedOnlyLocally = false) {
```

Comment:
@fabioh8010 Can you please change this to be a pure method where the `taskReport` is passed as a parameter? That will make the function easier to use.